test/e2e/mirror_test.go

```diff
@@ -186,3 +189,29 @@ func (tc *testContext) createMirrorSet() (*config.ImageTagMirrorSet, error) {
 	// Create the ImageTagMirrorSet in the cluster
 	return tc.client.Config.ConfigV1().ImageTagMirrorSets().Create(context.TODO(), itms, meta.CreateOptions{})
 }
+
+// testWorkloadWithMirroredImage runs a job using an image from a mirror registry URL
+func (tc *testContext) testWorkloadWithMirroredImage(t *testing.T) {
+	firstNodeAffinity, err := getAffinityForNode(&gc.allNodes()[0])
+	require.NoError(t, err, "could not get node affinity")
+	mirrorImage := tc.mirrorRegistry + "/powershell:lts-nanoserver-ltsc2022"
```

Comment:
we should choose this based on [Windows server version](https://github.com/openshift/windows-machine-config-operator/blob/master/test/e2e/network_test.go#L603)

here's a potential 1-liner
`mirrorImage := tc.mirrorRegistry + strings.Split(getWindowsServerContainerImage(), "/")[1]`